connectorconfig/applePay.go

```diff
@@ -0,0 +1,26 @@
+package connectorconfig
+
+type ApplePay struct {
+	// AppleMerchantIdentifier Merchant Identifier specified in the Apple Developer Merchant section
+	AppleMerchantIdentifier string `json:"appleMerchantIdentifier" yaml:"appleMerchantIdentifier"`
+	// AppleMerchantDisplayName Value to be displayed on the payment page
+	AppleMerchantDisplayName string `json:"appleMerchantDisplayName" yaml:"appleMerchantDisplayName" validate:"required_with=AppleMerchantIdentifier"`
+	// AppleMerchantCertificate Merchant certificate in the Apple Developer Merchant section (must be base64 encoded!)
+	AppleMerchantCertificate *string `json:"appleMerchantCertificate" yaml:"appleMerchantCertificate" validate:"required"`
```

Comment:
AppleMerchantCertificate and AppleMerchantPrivateKey need to be in secure fields